source/network/load-balancers/faqs.md

```diff
@@ -0,0 +1,79 @@
+# FAQs
+
+Here are some of the most frequently asked questions we get about our load balancers to help save you time.
+
+## How do I replace an expired certificate?
+
+This is covered in our [common changes](common-changes.html#replacing-an-expired-ssl-certificate) page.
+
+## How do I drain traffic from a particular target server?
+
+This is covered in our [common changes](common-changes.html#drain-traffic-from-a-particular-target-server) page.
+
+## How can I temporarily take a server out of load to run updates / maintenance?
+
+This is covered in our [common changes](common-changes.html#temporarily-remove-a-target-server-from-behind-the-load-balancer) page.
+
+## Why is all my traffic now coming from one IP?
+
+Once you've moved behind a load balancer, you may notice that some of your analytics on your backend servers, along with your logs appear to break. Where you used to see a breakdown of all the visitors to your site, you now just see one persistent visitor. The numbers won't have changed, but the IP address will have.
+
+This is down to the nature of the load balancer itself. Where visitors used to directly visit your server, they now visit the loadbalancer instead and it's the load balancer that makes requests to the backend server. As such, the only IP address you'll see in logs or analytics will likely be that of the load balancer.
+
+There's an easy fix. Whereas logs and analytics will usually derive the visitor IP from the requests source, they need to be directed to look at the standard `X-Forwarded-For` header instead. This is injected by the load balancer automatically, so should already be present.
```

Comment:
Is this true?

Reply:
I'd probably get this scrutinised by Dane before it gets merged in, just to ensure this is all still true.

Reply:
Yeah, this is injected automatically, however there are caveats:

* Works fine with HTTP connections
* Works fine with HTTPS connections that terminate at the LB or are re-encrypted by the LB before being passed onto the backend.
* Doesn't work with HTTPS passthrough (where the LB does no decryption)
* Doesn't work with TCP mode

A workaround for HTTPS passthrough or TCP mode is to enable PROXY protocol, but this requires support from the backend.